### ISSUE TITLE
Item Stats plugin not working in LMS/PvP Arena

### ISSUE THREAD

**Issue by Morcleon** [runelite/runelite#15332]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 10

### Bug description

No equipment stats are shown when hovering items in the LMS/PvP Arena loadout selection screens, just the name of the item and the action available.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/22901403/178771041-7c123f27-a5df-4491-86fd-39e887a65d00.png)

### RuneLite version

Runelite version: 1.8.26
Launcher version: 2.4.2

**Comment by swirle13** [comment]
gonna bump this because this has been around for years and still no fixes. Gonna maybe see if I can try my hand at opening a PR for this.

**Comment by swirle13** [comment]
After a few days, I got this fixed via creation of the necessary wiki pages that back the item stats cache that Runelite uses. Basically, the item stat overlay would look for an itemId that there was no wiki page for and so returned null, meaning no stat overlay was created. I went and wrote up some code and templates to find, assemble, and populate the data for the missing wiki pages into the wiki source code and manually created each of about ~50 pages for LMS variants of items that didn't exist.

All of those changes were just scraped and added to the items cache by @Nightfirecat via https://github.com/runelite/static.runelite.net/commit/93d6953fe7292b3a8cf452f3fadbcf430caafc75 and so should now be live in the game!
